.gitignore

```diff
@@ -18,3 +18,4 @@ public/styleguide/
 autoload_classmap.php
 .mysql
 .vagrant/
+public/img/captcha/
```

Comment:
This folder doesn't exist now. You probably should create it and then commit it and re-add it to the gitignore.